docs/api/puppeteer.frame.name.md

```diff
@@ -4,6 +4,15 @@ sidebar_label: Frame.name
 
 # Frame.name() method
 
+> Warning: This API is now obsolete.
+>
+> Use
+>
+> ```ts
+> const element = await frame.frameElement();
+> const name = await element.evaluate(frame => frame.name);
```

Comment:
Let's provide an example with the `id` as `frame.name()` would also return the Id of the element.